### PR TITLE
Update examples to use latest bevy

### PR DESCRIPTION
**Pull request by jameshiew**
When I tried running the examples last week they wouldn't compile due to some error with the version `syn` package that was being pulled in at the time, although this doesn't seem to be the case anymore. Updating the examples to bevy 0.7 and pinning dependencies at the time fixed the examples to work on my machine.

### CODE REVIEW

**Comment by bonsairobo** [comment]
Might as well upgrade to bevy 0.10

**Comment by jameshiew** [comment]
I tried updating the examples to bevy 0.10. Thanks for making this project!

File: examples-crate/Cargo.toml [hunk 1]
```diff
@@ -4,7 +4,12 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-bevy = { version = "0.7.0", features = ["bevy_winit", "render", "png", "x11"] }
+bevy = { version = "0.10.1", features = [
```
Comment (bonsairobo) [inline examples-crate/Cargo.toml:7]: Please just use "0.10" so we can automatically get patch updates.

**Review by bonsairobo — APPROVED** [review]

**Comment by bonsairobo** [comment]
We're up to bevy 0.13 now.

**Pull request closed without merging**